pkg/render/apiserver.go

```diff
@@ -2023,3 +2128,51 @@ func (c *apiServerComponent) getDeprecatedResources() []client.Object {
 
 	return renamedRscList
 }
+
+func (cfg *APIServerConfiguration) IsSidecarInjectionEnabled() bool {
+	return cfg.ApplicationLayer != nil &&
+		cfg.ApplicationLayer.Spec.SidecarInjection != nil &&
+		*cfg.ApplicationLayer.Spec.SidecarInjection == operatorv1.SidecarEnabled
+}
+
+func (c *apiServerComponent) l7AdmissionControllerContainer() corev1.Container {
+	volumeMounts := []corev1.VolumeMount{
+		c.cfg.TLSKeyPair.VolumeMount(c.SupportedOSType()),
+	}
+
+	l7AdmssCtrl := corev1.Container{
+		Name:            L7AdmissionControllerContainerName,
+		Image:           c.l7AdmissionControllerImage,
+		ImagePullPolicy: ImagePullPolicy(),
+		Env: []corev1.EnvVar{
+			corev1.EnvVar{
+				Name:  "L7ADMCTRL_TLSCERTPATH",
+				Value: c.cfg.TLSKeyPair.VolumeMountCertificateFilePath(),
+			},
+			corev1.EnvVar{
+				Name:  "L7ADMCTRL_TLSKEYPATH",
+				Value: c.cfg.TLSKeyPair.VolumeMountKeyFilePath(),
+			},
+			corev1.EnvVar{
```

Comment:
Just wanted to confirm - these are the only images that the admission controller is going to inject as sidecars right? 

Need to make sure that any images we deploy are configured via the operator (i.e., it would be an anti-pattern to have the admission controller make its own decisions about what images and versions to install).

Reply:
Part of me wonders if this admission controller should have been an endpoint on the tigera/operator, thinking about it a bit more. But it's definitely too late to make that call for this particular release.

Reply:
Yes... only this images. The sidecars are only two initContainers, the first one is dikastes that sets the iptables rules for the pod namespace, create the envoy configuration files and exits 0. The envoy is the an initContainer with the native sidecars k8s pattern. That will send the ext_authz and logs to the dikastes and l7-log-collector through unix sockets.